backend/lcfs/db/migrations/versions/2024-12-05-22-59_9206124a098b.py

```diff
@@ -0,0 +1,25 @@
+"""Add Organization to FSE
+
+Revision ID: 9206124a098b
+Revises: aeaa26f5cdd5
+Create Date: 2024-12-04 09:59:22.876386
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '9206124a098b'
+down_revision = '8491890dd688'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add the column 'organization' to 'final_supply_equipment' table
+    op.add_column("final_supply_equipment", sa.Column("organization", sa.String(), nullable=True))
```

Comment:
to reduce confusion we should name this field organization_name. just using organization is too close to referencing our org model.